src/main/java/com/sigma/cinematicketpro/exception/ResourceNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package com.sigma.cinematicketpro.exception;
+
+public class ResourceNotFoundException extends RuntimeException {
```

Comment:
This exception was removed since Spring Boot 2.0.0.RC1

https://github.com/spring-projects/spring-boot/commit/b98c7ed9f870ae7321cdbb6ea880052a6fca8a68#diff-92cf433adcf45d6e9e2a5e01fb4dedcf2ce946163b8a062d484a4371a9da7420